Source/WTF/wtf/text/WTFString.h

```diff
@@ -107,6 +107,7 @@ class String final {
 
     // Return characters8() or characters16() depending on CharacterType.
     template<typename CharacterType> const CharacterType* characters() const;
+    template<typename CharacterType> std::span<const CharacterType> span() const;
```

Comment:
Eventually we should make our classes consistent and remove synonyms. I wonder if we should standardize on `span<const LChar>`, `span<LChar>`, or `span8`?